packages/core/src/configuration/ConfigurationParser.ts

```diff
@@ -21,7 +25,33 @@ export class ConfigurationParser {
     this.configurations = config.services || [];
   }
 
-  public getConfig(type: ServiceType): ServiceConfiguration | undefined {
+  /**
+   * Get Metrics service configuration object
+   */
+  public getMetricsConfig(): ServiceConfiguration | undefined {
+    return this.getConfig(ServiceType.METRICS);
+  }
+
+  /**
+   * Get Keycloak service configuration object
+   */
+  public getKeycloakConfig(): ServiceConfiguration | undefined {
+    return this.getConfig(ServiceType.KEYCLOAK);
+  }
+
+  /**
+   * Get Push service configuration object
+   */
+  public getPushConfig(): ServiceConfiguration | undefined {
```

Comment:
I think those methods provides very little benefit, but cause big problem for maintaining.

Reply:
What problems you mean? This class has no functional value if we do not provide this methods. It is not doing any parsing. Helper methods are all that we could done here.

Reply:
Its functionality really is to abstract us from the JSON file itself, so that we should make it as implicit as possible, it is a helper class after all.

Reply:
The problem I have is that every time when a new service is added, should we add a new helper class here? If so it means we have to add new helper here (1 line of code), bump version and release a new version and maybe update all other modules to use this new version. It is quite a lot of maintenance work, but with not much benefit. It is enough to have the `getServiceConfig` method.

Reply:
@wei-lee That's awesome. We are going to update that then before we release this! Thank you so much for suggestion. It will be better for end users.

Reply:
Fair enough